File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGDisplay.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGDisplay {
    YGDisplayFlex,
    YGDisplayNone;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGDisplay forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGDisplay.kt
package app.cash.redwood.yoga.internal.enums

enum class YGDisplay {
    YGDisplayFlex, YGDisplayNone;

    fun getValue(): Int {
        return ordinal
    }

    companion object {
        fun forValue(value: Int): YGDisplay {
            return values()[value]
        }
    }
}
